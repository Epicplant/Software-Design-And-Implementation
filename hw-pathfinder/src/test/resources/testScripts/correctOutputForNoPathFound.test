# Tests that the output "no path found" is displayed when a path can't be attained between two nodes

# Creates a new graph
CreateGraph newGraph

# Adds node A to the graph
AddNode newGraph A

# Adds node B to the graph
AddNode newGraph B

# Adds node C to the graph
AddNode newGraph C

# Adds edge of weight 16 to the graph
AddEdge newGraph A B 16

# Finds the shortest path from node A to node C
FindPath newGraph A C

# Finds the shortest path from node C to node A
FindPath newGraph C A