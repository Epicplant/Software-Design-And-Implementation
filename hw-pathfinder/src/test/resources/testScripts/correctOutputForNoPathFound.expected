# Tests that the output "no path found" is displayed when a path can't be attained between two nodes

# Creates a new graph
created graph newGraph

# Adds node A to the graph
added node A to newGraph

# Adds node B to the graph
added node B to newGraph

# Adds node C to the graph
added node C to newGraph

# Adds edge of weight 16 to the graph
added edge 16.000 from A to B in newGraph

# Finds the shortest path from node A to node C
path from A to C:
no path found

# Finds the shortest path from node C to node A
path from C to A:
no path found